HW1/P2/P2b.py

```diff
@@ -1,3 +1,85 @@
 from P2 import *
+import findspark
+findspark.init()
+import pyspark
+import matplotlib.pyplot as plt
+import time
+ 
+if __name__ == "__main__":  
+    startTime=time.time()
+    
+    sc = pyspark.SparkContext()
 
-# Your code here
+    ran = range(0,2000, 1)
+    z = [(a,b) for a in ran for b in ran]
```

Comment:
The ranges and the cartesian product are done in python, but should be done in spark.